src/lsp/sql_parser/package.toml

```diff
@@ -0,0 +1,92 @@
+
+# name of package
+name = "sql_parser"
+skeleton = "library"
+
+# version if different from project version
+# version = "0.1.0"
+
+# synopsis if different from project synopsis
+# synopsis = ...
+
+# description if different from project description
+# description = ...
+
+# kind is either "library", "program" or "virtual"
+kind = "library"
+
+# authors if different from project authors
+# authors = [ "Me <me@metoo.org>" ]
+
+# name of a file to generate with the current version
+gen-version = "version.ml"
+
+# supported file generators are "ocamllex", "ocamlyacc" and "menhir" 
+# default is [ "ocamllex", "ocamlyacc" ] 
+generators = ["ocamllex"]
+
+# menhir options for the package
+#Example:
+#version = "2.0"
+#parser = { modules = ["parser"]; tokens = "Tokens" }
+#tokens = { modules = ["tokens"]}
+# menhir = ...
+
+# whether all modules should be packed/wrapped (default is true)
+# pack-modules = false
+
+# whether the package can be silently skipped if missing deps (default is false)
+# optional = true
+
+# module name used to pack modules (if pack-modules is true)
+# pack = "Mylib"
+
+# preprocessing options
+#  preprocess = "per-module (((action (run ./toto.sh %{input-file})) mod))" 
+preprocess = "pps ppx_deriving.show"
+
+# files to skip while updating at package level
+skip = ["main.ml", "index.mld"]
```

Comment:
I'd rather have `index.mld` generated using `synopsis` and `description` fields that ignored.

Reply:
I can't find any exemple it in the other package.toml ( the one in cobol_common, cobol_config, cobol_parser, cobol_preproc and ebcdic_lib all use `skip = ["main.ml", "index.mld"]`). If I try to do `drom project`, it change a lot of file and I don't know if it's safe to push